extensions/wikia/AdEngine/js/video/player/porvata/porvata.js

```diff
@@ -28,18 +33,37 @@ define('ext.wikia.adEngine.video.player.porvata', [
 			}).then(function (video) {
 				log(['porvata video player created', video], log.levels.debug, logGroup);
 
+				function inViewportCallback(isVisible) {
+					// Play video automatically only for the first time
+					if (isVisible && !autoPlayed && params.autoPlay) {
```

Comment:
Those conditions begs for unit testing :japanese_ogre: But it's kinda hard to unit test a function from this scope, I guess. Any ideas?